config/jobs/kubernetes/sig-cloud-provider/azure/release-master.yaml

```diff
@@ -299,7 +299,7 @@ periodics:
     workdir: false
   spec:
     containers:
-    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230719-72ce785784-master
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230703-e6ae5b372a-master
```

Comment:
these files are all auto generated from generate.sh so we should change the source image there and then run the script so this doesn't get overwritten by a later change (see the readme)

Reply:
Thanks for the clue! Updated.